message_ix/model/MACRO/macro_data_load.gms

```diff
@@ -231,7 +231,7 @@ LOOP(year_all $( ORD(year_all) > sum(year_all2$( macro_initial_period(year_all2)
 * new labor supply
    newlab(node_macro, year_all) = SUM(year_all2$( seq_period(year_all2,year_all) ), (labor(node_macro, year_all) - labor(node_macro, year_all2)*(1 - depr(node_macro))**duration_period(year_all))$((labor(node_macro, year_all) - labor(node_macro, year_all2)*(1 - depr(node_macro))**duration_period(year_all)) > 0)) + epsilon ;
 * calculation of utility discount factor based on discount rate (drate)
-   udf(node_macro, year_all)    = SUM(year_all2$( seq_period(year_all2,year_all) ), udf(node_macro, year_all2) * (1 - (DRATE(node_macro) - grow(node_macro, year_all)))**duration_period(year_all)) ;
+   udf(node_macro, year_all)    = SUM(year_all2$( seq_period(year_all2,year_all) ), udf(node_macro, year_all2) * (1 - (interestrate(year_all) - grow(node_macro, year_all)))**duration_period(year_all)) ;

```

Comment:
nice @volker-krey! I noticed as well that there are a few other places `drate` is used:

```
$ ack -i drate message_ix/model/MACRO/
message_ix/model/MACRO/macro_data_load.gms
63:* VK, 10 April 2008: DRATE (social discount rate, i.e. net rate of return on capital from MESSAGE) introduced as a new parameter as in MERGE 5
64:         drate(node)                 Social discount rate
134:$LOAD kpvs,kgdp,esub,depr,drate,lotol
233:* calculation of utility discount factor based on discount rate (drate)
262:* that are greater than drate = depr. To avoid negative values the absolute value is
266:finite_time_corr(node_macro, year) = abs(DRATE(node_macro) - grow(node_macro, year)) ;

message_ix/model/MACRO/macro_core.gms
142:* The utility discount rate for period :math:`y` is set to :math:`drate_{n} - grow_{n,y}`, where :math:`drate_{n}` is the discount rate used in MESSAGE, typically set to 5%,

message_ix/model/MACRO/macro_calibration.gms
86:* calculation of utility discount factor based on discount rate (drate)
87:   udf(node_macro, year_all)    = SUM(year_all2$( seq_period(year_all2,year_all) ), udf(node_macro, year_all2) * (1 - (DRATE(node_macro) - grow(node_macro, year_all)))**duration_period(year_all)) ;
91:finite_time_corr(node_macro, year) = abs(DRATE(node_macro) - grow(node_macro, year)) ;
```

Specifically lines 266 and 91 above. Should this also be changed?